src/dotnet/APIView/APIViewWeb/Models/ReviewRevisionModel.cs

```diff
@@ -38,5 +38,26 @@ public string Author
             get => _author ?? Review.Author;
             set => _author = value;
         }
+
+        [JsonIgnore]
+        public string DisplayName
```

Comment:
This is why schema-less is hard.

Reply:
You mean the ability to have versioned schemas?

Reply:
Not having fixed table structure and migrations like RDBMS